utilities/styles/nuisance_translate.json

```diff
@@ -158,14 +158,14 @@
     "resumFOScaleWSymDiff": "resum. FO scale W [diff.]",
     "resumFOScaleZSymAvg": "resum. FO scale Z [avg.]",
     "resumFOScaleZSymDiff": "resum. FO scale Z [diff.]",
-    "resumTNP_b_qqV": "resum. TNP B<sub>qqV</sub>",
-    "resumTNP_b_qg": "resum. TNP B<sub>qg</sub>",
-    "resumTNP_b_qqS": "resum. TNP B<sub>qqS</sub>",
-    "resumTNP_b_qqbarV": "resum. TNP B<sub>qqV̄</sub>",
-    "resumTNP_b_qqDS": "resum. TNP B<sub>qq</sub> DS",
-    "resumTNP_gamma_nu": "resum. TNP Γ<sub>ν</sub>",
-    "resumTNP_gamma_mu_q": "resum. TNP Γ<sub>μ,q</sub>",
-    "resumTNP_gamma_cusp": "resum. TNP Γ<sub>c,u,s,p</sub>",
-    "resumTNP_h_qqV": "resum. TNP H<sub>qqV</sub>",
-    "resumTNP_s": "resum. TNP S<sub>qqV</sub>"
+    "resumTNP_b_qqV": "resum. TNP BF qqV",
```

Comment:
Part of this is also used to create labels for nuisance parameters saved in the HEPData tables (although in plan Latex style, see https://github.com/WMass/WRemnants/blob/main/utilities/styles/nuisance_translate_hepdata.json).
 
Even if it is not mandatory it is useful to maintain a common convention to call these. Once the choice is final I'll remake the hepdata tables, just let me know.